Description: Add missing glyphs for Uighur language
Author: MihailJP <mihailjp@gmail.com>
Bug-Ubuntu: https://launchpad.net/bugs/943830

Index: ttf-dejavu-2.33/src/DejaVuSans-Bold.sfd
===================================================================
--- ttf-dejavu-2.33.orig/src/DejaVuSans-Bold.sfd	2011-02-27 15:37:20.000000000 +0100
+++ ttf-dejavu-2.33/src/DejaVuSans-Bold.sfd	2012-04-10 07:56:46.720120904 +0200
@@ -63034,6 +63034,42 @@
 Refer: 1608 1608 N 1 0 0 1 0 0 2
 Substitution2: "'fina' Terminal Forms in Arabic lookup 5" uniFBDA
 EndChar
+StartChar: uni06C7
+Encoding: 1735 1735 1735
+Width: 1273
+Flags: W
+AnchorPoint: "rtl-below" 625 -550 basechar 0
+AnchorPoint: "rtl-above" 625 1400 basechar 0
+LayerCount: 2
+Fore
+Refer: 1615 1615 N 1 0 0 1 55 -475 2
+Refer: 1608 1608 N 1 0 0 1 0 0 2
+Substitution2: "'fina' Terminal Forms in Arabic lookup 5" uniFBD8
+EndChar
+StartChar: uni06C8
+Encoding: 1736 1736 1736
+Width: 1273
+Flags: W
+AnchorPoint: "rtl-below" 625 -550 basechar 0
+AnchorPoint: "rtl-above" 625 1400 basechar 0
+LayerCount: 2
+Fore
+Refer: 1648 1648 N 1 0 0 1 110 -465 2
+Refer: 1608 1608 N 1 0 0 1 0 0 2
+Substitution2: "'fina' Terminal Forms in Arabic lookup 5" uniFBDC
+EndChar
+StartChar: uni06CB
+Encoding: 1739 1739 1739
+Width: 1273
+Flags: W
+AnchorPoint: "rtl-above" 650 1290 basechar 0
+AnchorPoint: "rtl-below" 625 -550 basechar 0
+LayerCount: 2
+Fore
+Refer: 1114126 -1 N 1 0 0 1 450 800 2
+Refer: 1608 1608 N 1 0 0 1 0 0 2
+Substitution2: "'fina' Terminal Forms in Arabic lookup 5" uniFBDF
+EndChar
 StartChar: uni06CC
 Encoding: 1740 1740 1740
 Width: 1877
@@ -63061,6 +63097,20 @@
 Substitution2: "'init' Initial Forms in Arabic lookup 7" uni06CE.init
 Substitution2: "'fina' Terminal Forms in Arabic lookup 5" uni06CE.fina
 EndChar
+StartChar: uni06D0
+Encoding: 1744 1744 1744
+Width: 1877
+Flags: W
+AnchorPoint: "rtl-below" 851 -850 basechar 0
+AnchorPoint: "rtl-above" 550 850 basechar 0
+LayerCount: 2
+Fore
+Refer: 1114128 -1 N 1 0 0 1 780 -500 2
+Refer: 1609 1609 N 1 0 0 1 0 3.05176e-05 2
+Substitution2: "'init' Initial Forms in Arabic lookup 7" uniFBE6
+Substitution2: "'medi' Medial Forms in Arabic lookup 6" uniFBE7
+Substitution2: "'fina' Terminal Forms in Arabic lookup 5" uniFBE5
+EndChar
 StartChar: uni06D5
 Encoding: 1749 1749 1749
 Width: 1209
@@ -154496,6 +154546,28 @@
 Refer: 65244 65244 N 1 0 0 1 0 0 2
 Refer: 1114126 -1 N 1 0 0 1 75 1500 2
 EndChar
+StartChar: uniFBD7
+Encoding: 64471 64471 64471
+Width: 1273
+Flags: W
+AnchorPoint: "rtl-below" 625 -550 basechar 0
+AnchorPoint: "rtl-above" 625 1400 basechar 0
+LayerCount: 2
+Fore
+Refer: 1608 1608 N 1 0 0 1 0 0 2
+Refer: 1615 1615 N 1 0 0 1 55 -475 2
+EndChar
+StartChar: uniFBD8
+Encoding: 64472 64472 64472
+Width: 1284
+Flags: W
+AnchorPoint: "rtl-below" 625 -550 basechar 0
+AnchorPoint: "rtl-above" 625 1400 basechar 0
+LayerCount: 2
+Fore
+Refer: 65262 65262 N 1 0 0 1 0 0 2
+Refer: 1615 1615 N 1 0 0 1 55 -475 2
+EndChar
 StartChar: uniFBD9
 Encoding: 64473 64473 64473
 Width: 1273
@@ -154518,6 +154590,94 @@
 Refer: 1626 1626 N 1 0 0 1 113 -450 2
 Refer: 65262 65262 N 1 0 0 1 0 0 2
 EndChar
+StartChar: uniFBDB
+Encoding: 64475 64475 64475
+Width: 1273
+Flags: W
+AnchorPoint: "rtl-below" 625 -550 basechar 0
+AnchorPoint: "rtl-above" 625 1400 basechar 0
+LayerCount: 2
+Fore
+Refer: 1608 1608 N 1 0 0 1 0 0 2
+Refer: 1648 1648 N 1 0 0 1 110 -465 2
+EndChar
+StartChar: uniFBDC
+Encoding: 64476 64476 64476
+Width: 1284
+Flags: W
+AnchorPoint: "rtl-below" 625 -550 basechar 0
+AnchorPoint: "rtl-above" 625 1400 basechar 0
+LayerCount: 2
+Fore
+Refer: 65262 65262 N 1 0 0 1 0 0 2
+Refer: 1648 1648 N 1 0 0 1 110 -465 2
+EndChar
+StartChar: uniFBDE
+Encoding: 64478 64478 64478
+Width: 1273
+Flags: W
+AnchorPoint: "rtl-above" 650 1290 basechar 0
+AnchorPoint: "rtl-below" 625 -550 basechar 0
+LayerCount: 2
+Fore
+Refer: 1608 1608 N 1 0 0 1 0 0 2
+Refer: 1114126 -1 N 1 0 0 1 450 800 2
+EndChar
+StartChar: uniFBDF
+Encoding: 64479 64479 64479
+Width: 1284
+Flags: W
+AnchorPoint: "rtl-above" 650 1290 basechar 0
+AnchorPoint: "rtl-below" 625 -550 basechar 0
+LayerCount: 2
+Fore
+Refer: 65262 65262 N 1 0 0 1 0 0 2
+Refer: 1114126 -1 N 1 0 0 1 450 800 2
+EndChar
+StartChar: uniFBE4
+Encoding: 64484 64484 64484
+Width: 1877
+Flags: W
+AnchorPoint: "rtl-below" 851 -850 basechar 0
+AnchorPoint: "rtl-above" 550 850 basechar 0
+LayerCount: 2
+Fore
+Refer: 1609 1609 N 1 0 0 1 0 3.05176e-05 2
+Refer: 1114128 -1 N 1 0 0 1 780 -500 2
+EndChar
+StartChar: uniFBE5
+Encoding: 64485 64485 64485
+Width: 2072
+Flags: W
+AnchorPoint: "rtl-below" 851 -860 basechar 0
+AnchorPoint: "rtl-above" 750 700 basechar 0
+LayerCount: 2
+Fore
+Refer: 1114128 -1 N 1 0 0 1 780 -510 2
+Refer: 65264 65264 N 1 0 0 1 0 0 2
+EndChar
+StartChar: uniFBE6
+Encoding: 64486 64486 64486
+Width: 769
+Flags: W
+AnchorPoint: "rtl-below" 418 -550 basechar 0
+AnchorPoint: "rtl-above" 418 1000 basechar 0
+LayerCount: 2
+Fore
+Refer: 1114131 -1 N 1 0 0 1 0 0 2
+Refer: 1114128 -1 N 1 0 0 1 338 -250 2
+EndChar
+StartChar: uniFBE7
+Encoding: 64487 64487 64487
+Width: 836
+Flags: W
+AnchorPoint: "rtl-below" 418 -550 basechar 0
+AnchorPoint: "rtl-above" 418 1000 basechar 0
+LayerCount: 2
+Fore
+Refer: 1114132 -1 N 1 0 0 1 0 0 2
+Refer: 1114128 -1 N 1 0 0 1 338 -250 2
+EndChar
 StartChar: uniFBE8
 Encoding: 64488 64488 64488
 Width: 769
Index: ttf-dejavu-2.33/src/DejaVuSans.sfd
===================================================================
--- ttf-dejavu-2.33.orig/src/DejaVuSans.sfd	2011-02-27 15:37:21.000000000 +0100
+++ ttf-dejavu-2.33/src/DejaVuSans.sfd	2012-04-10 07:56:46.752120905 +0200
@@ -81028,6 +81028,42 @@
 Refer: 1608 1608 N 1 0 0 1 0 0 2
 Substitution2: "'fina' Terminal Forms in Arabic lookup 5" uniFBDA
 EndChar
+StartChar: uni06C7
+Encoding: 1735 1735 1735
+Width: 989
+Flags: W
+AnchorPoint: "rtl-above" 500 1350 basechar 0
+AnchorPoint: "rtl-below" 500 -500 basechar 0
+LayerCount: 2
+Fore
+Refer: 1615 1615 N 1 0 0 1 -92 -552 2
+Refer: 1608 1608 N 1 0 0 1 0 0 2
+Substitution2: "'fina' Terminal Forms in Arabic lookup 5" uniFBD8
+EndChar
+StartChar: uni06C8
+Encoding: 1736 1736 1736
+Width: 989
+Flags: W
+AnchorPoint: "rtl-above" 500 1450 basechar 0
+AnchorPoint: "rtl-below" 500 -500 basechar 0
+LayerCount: 2
+Fore
+Refer: 1648 1648 N 1 0 0 1 -20 -485 2
+Refer: 1608 1608 N 1 0 0 1 0 0 2
+Substitution2: "'fina' Terminal Forms in Arabic lookup 5" uniFBDC
+EndChar
+StartChar: uni06CB
+Encoding: 1739 1739 1739
+Width: 989
+Flags: W
+AnchorPoint: "rtl-above" 500 1250 basechar 0
+AnchorPoint: "rtl-below" 500 -500 basechar 0
+LayerCount: 2
+Fore
+Refer: 1114127 -1 N 1 0 0 1 300 800 2
+Refer: 1608 1608 N 1 0 0 1 0 0 2
+Substitution2: "'fina' Terminal Forms in Arabic lookup 5" uniFBDF
+EndChar
 StartChar: uni06CC
 Encoding: 1740 1740 1740
 Width: 1603
@@ -81055,6 +81091,20 @@
 Substitution2: "'init' Initial Forms in Arabic lookup 7" uni06CE.init
 Substitution2: "'fina' Terminal Forms in Arabic lookup 5" uni06CE.fina
 EndChar
+StartChar: uni06D0
+Encoding: 1744 1744 1744
+Width: 1603
+Flags: W
+AnchorPoint: "rtl-below" 725 -870 basechar 0
+AnchorPoint: "rtl-above" 500 750 basechar 0
+LayerCount: 2
+Fore
+Refer: 1114129 -1 N 1 0 0 1 650 -520 2
+Refer: 1609 1609 N 1 0 0 1 0 0 2
+Substitution2: "'init' Initial Forms in Arabic lookup 7" uniFBE6
+Substitution2: "'medi' Medial Forms in Arabic lookup 6" uniFBE7
+Substitution2: "'fina' Terminal Forms in Arabic lookup 5" uniFBE5
+EndChar
 StartChar: uni06D5
 Encoding: 1749 1749 1749
 Width: 1073
@@ -189419,6 +189469,28 @@
 Refer: 65244 65244 N 1 0 0 1 0 0 2
 Refer: 1114127 -1 N 1 0 0 1 75 1500 2
 EndChar
+StartChar: uniFBD7
+Encoding: 64471 64471 64471
+Width: 989
+Flags: W
+AnchorPoint: "rtl-above" 500 1350 basechar 0
+AnchorPoint: "rtl-below" 500 -500 basechar 0
+LayerCount: 2
+Fore
+Refer: 1608 1608 N 1 0 0 1 0 0 2
+Refer: 1615 1615 N 1 0 0 1 -92 -552 2
+EndChar
+StartChar: uniFBD8
+Encoding: 64472 64472 64472
+Width: 1058
+Flags: W
+AnchorPoint: "rtl-above" 500 1350 basechar 0
+AnchorPoint: "rtl-below" 500 -500 basechar 0
+LayerCount: 2
+Fore
+Refer: 65262 65262 N 1 0 0 1 0 0 2
+Refer: 1615 1615 N 1 0 0 1 -92 -552 2
+EndChar
 StartChar: uniFBD9
 Encoding: 64473 64473 64473
 Width: 989
@@ -189441,6 +189513,94 @@
 Refer: 1626 1626 N 1 0 0 1 -12 -450 2
 Refer: 65262 65262 N 1 0 0 1 0 0 2
 EndChar
+StartChar: uniFBDB
+Encoding: 64475 64475 64475
+Width: 989
+Flags: W
+AnchorPoint: "rtl-above" 500 1450 basechar 0
+AnchorPoint: "rtl-below" 500 -500 basechar 0
+LayerCount: 2
+Fore
+Refer: 1608 1608 N 1 0 0 1 0 0 2
+Refer: 1648 1648 N 1 0 0 1 -20 -485 2
+EndChar
+StartChar: uniFBDC
+Encoding: 64476 64476 64476
+Width: 1058
+Flags: W
+AnchorPoint: "rtl-above" 500 1450 basechar 0
+AnchorPoint: "rtl-below" 500 -500 basechar 0
+LayerCount: 2
+Fore
+Refer: 65262 65262 N 1 0 0 1 0 0 2
+Refer: 1648 1648 N 1 0 0 1 -20 -485 2
+EndChar
+StartChar: uniFBDE
+Encoding: 64478 64478 64478
+Width: 989
+Flags: W
+AnchorPoint: "rtl-above" 500 1250 basechar 0
+AnchorPoint: "rtl-below" 500 -500 basechar 0
+LayerCount: 2
+Fore
+Refer: 1608 1608 N 1 0 0 1 0 0 2
+Refer: 1114127 -1 N 1 0 0 1 300 800 2
+EndChar
+StartChar: uniFBDF
+Encoding: 64479 64479 64479
+Width: 1058
+Flags: W
+AnchorPoint: "rtl-above" 500 1250 basechar 0
+AnchorPoint: "rtl-below" 500 -500 basechar 0
+LayerCount: 2
+Fore
+Refer: 65262 65262 N 1 0 0 1 0 0 2
+Refer: 1114127 -1 N 1 0 0 1 300 800 2
+EndChar
+StartChar: uniFBE4
+Encoding: 64484 64484 64484
+Width: 1603
+Flags: W
+AnchorPoint: "rtl-below" 725 -870 basechar 0
+AnchorPoint: "rtl-above" 500 750 basechar 0
+LayerCount: 2
+Fore
+Refer: 1609 1609 N 1 0 0 1 0 0 2
+Refer: 1114129 -1 N 1 0 0 1 650 -520 2
+EndChar
+StartChar: uniFBE5
+Encoding: 64485 64485 64485
+Width: 1707
+Flags: W
+AnchorPoint: "rtl-below" 725 -870 basechar 0
+AnchorPoint: "rtl-above" 600 750 basechar 0
+LayerCount: 2
+Fore
+Refer: 1114129 -1 N 1 0 0 1 650 -520 2
+Refer: 65264 65264 N 1 0 0 1 0 0 2
+EndChar
+StartChar: uniFBE6
+Encoding: 64486 64486 64486
+Width: 570
+Flags: W
+AnchorPoint: "rtl-below" 299 -550 basechar 0
+AnchorPoint: "rtl-above" 299 1000 basechar 0
+LayerCount: 2
+Fore
+Refer: 1114129 -1 N 1 0 0 1 224 -250 2
+Refer: 1114132 -1 N 1 0 0 1 0 0 2
+EndChar
+StartChar: uniFBE7
+Encoding: 64487 64487 64487
+Width: 618
+Flags: W
+AnchorPoint: "rtl-below" 299 -550 basechar 0
+AnchorPoint: "rtl-above" 299 1000 basechar 0
+LayerCount: 2
+Fore
+Refer: 1114133 -1 N 1 0 0 1 0 0 2
+Refer: 1114129 -1 N 1 0 0 1 224 -250 2
+EndChar
 StartChar: uniFBE8
 Encoding: 64488 64488 64488
 Width: 570
